apps/docs/content/components/Image.mdx

```diff
@@ -0,0 +1,94 @@
+---
+title: Image
+status: Experimental
+source: https://github.com/primer/brand/blob/main/packages/react/src/Image/Image.tsx
+storybook: '/brand/storybook/?path=/docs/components-image--playground'
+description: Use Image to create both normal and responsive image elements.
+---
+
+import {Label} from '@primer/react'
+
+```js
+import {Image} from '@primer/react-brand'
+```
+
+## Examples
+
+### Default
+
+This component uses the `img` element by default.
+
+```jsx live
+<Image
+  src="https://via.placeholder.com/600x400/d3d9df/d3d9df.png"
+  alt="placeholder, blank area with an off-white background color"
+/>
+```
+
```

Comment:
Could we add an additional example using `srcSet` on an `img` please? That would be a super-helpful for copy-pasting.

Reply:
This has been added